src/utils/daos.c

```diff
@@ -1280,6 +1384,19 @@ help_hdlr(int argc, char *argv[], struct cmd_args_s *ap)
 
 	if (argc <= 2) {
 		FIRST_LEVEL_HELP();
+	} else if (strcmp(argv[2], "filesystem") != 0) {
+		if (strcmp(argv[3], "copy") == 0) {
```

Comment:
The equality is reversed, we need to protect against the case where another argument isn't passed at all, and we should probably handle the shorthand "fs". 
Consider these cases:
`daos help filesystem`
`daos help fs`
```suggestion
	} else if (strcmp(argv[2], "filesystem") == 0 || strcmp(argv[2], "fs") == 0) {
		if (argc == 3) {
            		ALL_FS_CMDS_HELP();
		} else if (strcmp(argv[3], "copy") == 0) {
```